src/main/webapp/WEB-INF/views/car/add.jsp

```diff
@@ -0,0 +1,20 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Add car</title>
+</head>
+<body>
+<h1>Please, fill in the car info.</h1>
+
+<h4 style="color: red">${errorMessage}</h4>
```

Comment:
probably we don't need this, same for other JSPs

Reply:
Now we don't